modules/tutorials/pages/accumulators-tutorial.adoc

```diff
@@ -46,8 +46,8 @@ As Figure 2 shows, Dan has 4 direct friends -- Tom, Kevin, Jenny, and Nancy, eac
 
 ACCUM and POST-ACCUM clauses are computed in stages, where-in a SELECT-FROM-WHERE query block, ACCUM is executed first, followed by the POST-ACCUM clause.
 
-* *ACCUM* executes its statement(s) once for each matched edge (or path) of the FROM clause pattern. Further, ACCUM parallelly executes its statements for all the matches.
-* *POST-ACCUM* executes its statement(s) once for each involved vertex. Note that each statement within the POST-ACCUM clause can refer to either source vertices or target vertices but not both. Its statements can access the aggregated accumulator result computed in the ACCUM clause.
+* *ACCUM* executes its statement(s) once for each matched subgraph specified by the FROM, WHERE, and PER clauses. Further, ACCUM parallelly executes its statements for all the matches.
+* *POST-ACCUM* executes its statement(s) once for each involved vertex. Note that each POST-ACCUM clause should bind to one and only one vertex alias specified in the FROM clause. Its statements can access the aggregated accumulator result computed in the ACCUM clause.
```

Comment:
* *POST-ACCUM* executes its statement(s) per binding vertex instance. Note that each POST-ACCUM clause must bind 1 vertex alias appeared in the FROM clause; at least 1 statement in a POST-ACCUM clause have the binding vertex alias. POST-ACCUM statements can access the aggregated accumulator result computed in the ACCUM clause.